### ISSUE TITLE
unable to view full alt text in recent versions

### ISSUE THREAD

**Issue by trwnh** [tateisu/SubwayTooter#238]
tapping the caption on an attachment used to open a modal with the full alt text. now, it does not do that. when i tap on the alt text preview, it gets highlighted and then stuck in a highlighted state.

Sony Xperia 1 III
Android 12
